packages/postgres/src/default-pool.ts

```diff
@@ -1,7 +1,8 @@
-import { type CursorIterator, PostgresPool, type QueryParams } from './pool.js';
+import { PostgresPool } from './pool.js';
+export { type CursorIterator, type QueryParams } from './pool.js';
```

Comment:
This wouldn't be handled by `ignoreUsedVariables`. Maybe I'm just not a fan of this rule? I sort of feel like splitting up the exports makes things harder to comprehend.

Reply:
I'm fine disabling this rule.